### ISSUE TITLE
Current version of plugin is incompatible with last (1.6) version of sylius.

### ISSUE THREAD

**Issue by andreymeretsky** [BitBagCommerce/SyliusCmsPlugin#286]
composer require bitbag/cms-plugin
Using version ^2.3 for bitbag/cms-plugin
./composer.json has been updated
Loading composer repositories with package information
Updating dependencies (including require-dev)
Your requirements could not be resolved to an installable set of packages.

  Problem 1
    - bitbag/cms-plugin v2.3.1 requires sylius/sylius ~1.4.0|~1.5.0 -> satisfiable by sylius/sylius[1.4.x-dev, 1.5.x-dev, v1.4.0, v1.4.0-BETA.1, v1.4.1, v1.4.10, v1.4.11, v1.4.2, v1.4.3, v1.4.4, v1.4.5, v1.4.6, v1.4.7, v1.4.8, v1.4.9, v1.5.0, v1.5.0-RC.1, v1.5.1, v1.5.2, v1.5.3, v1.5.4, v1.5.5, v1.5.6, v1.5.7, v1.5.8] but these conflict with your requirements or minimum-stability.
    - bitbag/cms-plugin v2.3.0 requires sylius/sylius ~1.4.0 -> satisfiable by sylius/sylius[1.4.x-dev, v1.4.0, v1.4.0-BETA.1, v1.4.1, v1.4.10, v1.4.11, v1.4.2, v1.4.3, v1.4.4, v1.4.5, v1.4.6, v1.4.7, v1.4.8, v1.4.9] but these conflict with your requirements or minimum-stability.
    - Conclusion: don't install bitbag/cms-plugin v2.3.3
    - Conclusion: remove friendsofsymfony/ckeditor-bundle 2.2.0
    - Installation request for bitbag/cms-plugin ^2.3 -> satisfiable by bitbag/cms-plugin[v2.3.0, v2.3.1, v2.3.2, v2.3.3].
    - Conclusion: don't install friendsofsymfony/ckeditor-bundle 2.2.0
    - bitbag/cms-plugin v2.3.2 requires friendsofsymfony/ckeditor-bundle ^1.1 -> satisfiable by friendsofsymfony/ckeditor-bundle[1.1.0, 1.2.0].
    - Can only install one of: friendsofsymfony/ckeditor-bundle[1.1.0, 2.2.0].
    - Can only install one of: friendsofsymfony/ckeditor-bundle[1.2.0, 2.2.0].
    - Installation request for friendsofsymfony/ckeditor-bundle (locked at 2.2.0) -> satisfiable by friendsofsymfony/ckeditor-bundle[2.2.0].

**Comment by stefandoorn** [comment]
It looks to me you are using CKEditor ^2.0, but looking at the composer.json of this repo it's not supported. Just submitted #292 as I was having similar issue.

**Comment by Lowcost-dev** [comment]
Else there's a way to handle this, just add into your composer.json :
```
...
"require": {
        ...
        "friendsofsymfony/ckeditor-bundle": "**2.2 as 1.1**",
        ...
},
...
```
You can find the detailed anwser [here](https://stackoverflow.com/questions/21052831/how-to-solve-two-packages-requirements-conflicts-when-running-composer-install/21467982#21467982)

**Comment by patrick477** [comment]
we've added a fix in v2.3.5
